### DISCUSSION TITLE
Updating nextjs to 13.5 breaks the Application

### DISCUSSION THREAD

**Discussion by Zaid-maker** [vercel/next.js#55826] [Help]
### Summary

Currently using Next@13.4.19 and when i update it to latest it breaks my application and nothing showing on main page.

Below is the screenshot
![image](https://github.com/vercel/next.js/assets/53424436/b38bc796-a100-4482-a633-0d41167be569)

rollbacking the versions fix the application

![image](https://github.com/vercel/next.js/assets/53424436/5064c7bb-057b-4351-bfb7-414c45dd09c2)

### Additional information

```js
Operating System:
      Platform: linux
      Arch: x64
      Version: #202308111259 SMP PREEMPT_DYNAMIC Fri Aug 11 14:20:00 UTC 2023
    Binaries:
      Node: 18.17.1
      npm: 9.6.7
      Yarn: 1.22.19
      pnpm: 8.7.5
    Relevant Packages:
      next: 13.4.19
      eslint-config-next: 13.4.19
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.2.2
    Next.js Config:
      output: N/A
```

**Comment by devjiwonchoi** [comment]
Have you tried to delete `.next` and `node_modules` directories, reinstall the dependencies and run `next dev`?
If so, could you provide a repro?

P.S. Your additional info says `13.4.19`.

**Reply by Zaid-maker** [reply]
Can u help please?

**Comment by Zaid-maker** [comment]
> Have you tried to delete `.next` and `node_modules` directories, reinstall the dependencies and run `next dev`? If so, could you provide a repro?
> 
> P.S. Your additional info says `13.4.19`.

I didn't try deleting the folders but I have a preview PR open and it's still breaks my application

Edit: even after deleting `.next` and node_modules still breaks the app :(
